decidim-assemblies/app/models/decidim/assembly.rb

```diff
@@ -47,5 +56,17 @@ def hashtag
     def to_param
       slug
     end
+
+    def private_space?
+      private_space
+    end
+
+    def self.private_space
```

Comment:
`def private_assemblies` maybe? So it doesn't clash with meaning with `Decidim::Participable.public_spaces`...